### PR TITLE
Features/transaction generation

### PR DESCRIPTION
**Pull request by ptelman**

### CODE REVIEW

**Comment by ptelman** [comment]
We should store somewhere list of potential recipients (now I take random from the ones existing in previous transactions which will not work from the beginning when blockchain is empty).

File: PutCoin/Models/Transaction.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            {
+                Destinations = designations,
+                Id = Guid.NewGuid(),
+                OriginTransactionIds = new[] { originTransaction.Id },
+                Signature = initiator.Signature,
+                User = initiator
+            };
+        }
+
+        private static List<TransactionDestination> GetDesignationsForNewTransaction(decimal valueToSpend, Random random, User[] receipents)
```
Comment (mpopiol) [inline PutCoin/Models/Transaction.cs]: any reason for passing same random? why not just use new ones?

**Review by mpopiol — COMMENTED** [review]
I don't get one thing but it seems good overall

File: PutCoin/Models/Transaction.cs (hunk 1 above)
Comment (ptelman) [reply PutCoin/Models/Transaction.cs]: I don't get it. If you mean if passing Random object is needed - probably not and there could be just int passed.

Comment (ptelman) [reply PutCoin/Models/Transaction.cs]: Done

**Pull request merged**